src/views/shared/SecurityBeacon.js

```diff
@@ -123,21 +123,19 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
       // Animate loading the security beacon with a loading bar for the border
       // This occurrs when the username has been checked against Okta.
       border.removeClass('auth-beacon-border');
-      Animations.radialProgressBar({
+      await Animations.radialProgressBar({
         $el: radialProgressBar,
         swap: function () {
           image.fadeOut(duration, function () {
```

Comment:
Since we're modifying this function, might be nice to demonstrate some other es6 additions:

```javascript
swap() {
  image.fadeOut(duration, () => {
            setBackgroundImage(image, appState);
            image.fadeIn(duration);
  });
}
```